src/code/irqmgr.c

```diff
@@ -210,23 +272,25 @@ void IrqMgr_ThreadEntry(void* arg0) {
         }
     }
 
-    osSyncPrintf("ＩＲＱマネージャスレッド実行終了\n"); // "End of IRQ manager thread execution"
+    // "End of IRQ manager thread execution"
+    osSyncPrintf("ＩＲＱマネージャスレッド実行終了\n");
 }
 
 void IrqMgr_Init(IrqMgr* this, void* stack, OSPri pri, u8 retraceCount) {
     LogUtils_CheckNullPointer("this", this, "../irqmgr.c", 346);
     LogUtils_CheckNullPointer("stack", stack, "../irqmgr.c", 347);
 
     this->clients = NULL;
+    // Messages to send to each client message queue on each interrupt event
     this->retraceMsg.type = OS_SC_RETRACE_MSG;
     this->prenmiMsg.type = OS_SC_PRE_NMI_MSG;
     this->nmiMsg.type = OS_SC_NMI_MSG;
-    this->resetStatus = STATUS_IDLE;
+    this->resetStatus = IRQ_RESET_IDLE;
     this->resetTime = 0;
 
     osCreateMesgQueue(&this->queue, this->msgBuf, ARRAY_COUNT(this->msgBuf));
     osSetEventMesg(OS_EVENT_PRENMI, &this->queue, (OSMesg)PRE_NMI_MSG);
     osViSetEvent(&this->queue, (OSMesg)RETRACE_MSG, retraceCount);
-    osCreateThread(&this->thread, 0x13, IrqMgr_ThreadEntry, this, stack, pri);
+    osCreateThread(&this->thread, 19, IrqMgr_ThreadEntry, this, stack, pri);
```

Comment:
Will threads get defines for their ids, or is it not standard enough to be worth it?

Reply:
Different threads may have the same thread id, since it's only relevant for debugging afaik. I think I will make thread id defines in the future anyway, but probably not in this PR

Reply:
From my documentation, only the os internal threads (vi/pi) use the same thread id.